### DISCUSSION TITLE
Page numbering - strange behaviour

### DISCUSSION THREAD

**Discussion by giwul70** [sumatrapdfreader/sumatrapdf#3742] [General]
Although I was at the end of the file, the page numbering showed 79/83. 
Noticed that in this file only.. at a specific page (page 15), there was no page number (i.e.  [ ___/83] and the next page, where is should have read 16, it skipped back to 12 and the page numbering was 16/83.

I do not want to share the file, it contains personal info. I did create a short video though.

It is only with this specific file and on this specific page, so it is a minor issue, but would nonetheless like to report it.
I _believe_... the reason may be that a part of  the text on that page was/is rotated (180 degr).
Source: scanned pages, converted to a single PDF.

See atached video, page #15.

https://github.com/sumatrapdfreader/sumatrapdf/assets/101186455/de5c889b-773e-4952-a808-ab11d9f44bf2

**Comment by GitHubRulesOK** [comment]
Numbers are either provided by a document or not. Hence the two indicators the (1/83) is the relative page so should always count 13/83 14/83 15/83 what may be odd to some people is that internally PDF considers first page as 0 and 2nd page as 1 etc. and the counter is using page# +1 as a default counter, the numbers in the rectangle are page labels and when custom the counter and custom labels are shown so #0 may be  "Cover" and custom page #1 shown as (2/##) etc. So labels can be any sequence set by the author such as A B C or 1, 1.1, 1.1.1 (that special case can causes problems and there is one open issue)

The Author may have intentionally or accidentally changed or merged different labels.

**Comment by giwul70** [comment]
As said, these were simple scans. Scans #1, #2, #3 and so on as jpg files. Then select all and combine them into 1 PDF using Acrobat.
The numbering is fine uptil and incl. 14, page 15 has a blank 'box' and page 16 goes back to page 12 (in the box).
Anyway, as said, minor thing only.

I am using SumatraPDF as default. It is great, fast and more user friendly than Acrobat.
(Only for creating / combining PDFs, I use a licensed Acrobat)

![SnagIt-03102023 110237](https://github.com/sumatrapdfreader/sumatrapdf/assets/101186455/1b497045-8fb5-4848-99fb-9c52018bb59f)

**Reply by GitHubRulesOK** [reply]
can only see what a sample does so either emulate by scanning 16 blank pages or send a copy to developer for analysis. there should be a means in licensed acrobat to check if "page labels" are  shown in sequence since pages stored in a PDF do not have to be same order as displayed.
The author decides here fred had a brainstorm while drinking coffe as he intended a count down and forgot to write 2 for page 3
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/eedb98ae-26a3-4206-90b8-8107cb474fe8)


he could try to correct numbering 3 as 2 and damage 2
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/517a97dd-cb4a-498d-ab69-e7f9852c4c75)
